README.md

```diff
@@ -367,8 +371,8 @@ spec:
   name: "shared-example-node-workload"
   allCores: true
   reservedCPUs:
```

Comment:
Thanks for making this change to allow a powerprofile to be configured for the reserved CPUs. Did you implement this change in a backwards compatible manner? I'm just wondering what will happen if a user has created a shared PowerWorkload with a previous version of KPM and then the KPM operator is upgraded. What is going to happen when the CR with the old contents of the "reservedCPUs" is parsed?

Reply:
It will show up as invalid and won't be installed. To mitigate this users will have to tweak their workload manifest. I added a note in the README specifically about this and will add it to this version's release notes.
I'll add it to the fixes list for the upcoming release

Reply:
Shouldn't this be fixed now? Why would you release a new version of KPM that breaks the existing users? That doesn't seem acceptable.